### PR TITLE
dark mode

### PR DESCRIPTION
**Pull request by mjoyce91**
Oct-03-2019 21-26-35

### CODE REVIEW

File: src/Containers/DarkMode/DarkMode.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
```
Comment (burgwyn) [inline src/Containers/DarkMode/DarkMode.jsx]: Test(s)

File: src/Components/AccountDropdown/DarkModeToggle.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
```
Comment (burgwyn) [inline src/Components/AccountDropdown/DarkModeToggle.jsx]: Test(s)

File: package.json [hunk 3]
```diff
@@ -105,6 +106,7 @@
     "node-mocks-http": "^1.7.6",
     "node-sass": "^4.7.2",
     "object-assign": "4.1.1",
+    "object.entries": "^1.1.0",
```
Comment (burgwyn) [inline package.json]: Are we using this?

Comment (mjoyce91) [reply package.json]: The `darkreader` library uses it, and apparently you have to polyfill it when using Jest.

**Comment by mjoyce91** [comment]
I'm going to add a new `personalization` feature flag to wrap this in

**Comment by mjoyce91** [comment]
@burgwyn see updates

**Pull request merged**